schema.sql

```diff
@@ -1,11 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    id INTEGER,
+    id SERIAL PRIMARY KEY,
     name VARCHAR(255),
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg DECIMAL(8, 2)
+    weight_kg DECIMAL(8, 2),
+    species VARCHAR(255)
```

Comment:
- [ ] Well, you are asked to add a column to a table, and you do but in an unprofessional way, we are expecting you to use appropriate to add something to a given table, try reading on the `ALTER` command and update your project with the new info you will gain